sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EpidemiologicalDataCasePage.java

```diff
@@ -0,0 +1,107 @@
+  public static final By EXPOSURE_REGION = By.cssSelector(".v-window #region");
+  public static final By DISTRICT = By.cssSelector(".v-window #district");
+  public static final By COMMUNITY = By.cssSelector(".v-window #community");
+  public static final By STREET = By.cssSelector(".v-window #street");
+  public static final By HOUSE_NUMBER = By.cssSelector(".v-window #houseNumber");
+  public static final By ADDITIONAL_INFORMATION =
+      By.cssSelector(".v-window #additionalInformation");
+  public static final By POSTAL_CODE = By.cssSelector(".v-window #postalCode");
+  public static final By CITY = By.cssSelector(".v-window #city");
+  public static final By AREA_TYPE = By.cssSelector(".v-window #areaType");
+  public static final By COMMUNITY_CONTACT_PERSON = By.cssSelector(".v-window #details");
+  public static final By DONE_BUTTON = By.cssSelector(".v-window #commit");
+  public static final By DISCARD_BUTTON = By.cssSelector(".v-window #discard");
+
+  public static final By ACTIVITY_DETAILS_KNOWN =
+      By.cssSelector("#activityAsCaseDetailsKnown span:nth-child(1)v");
+  public static final By ACTIVITY_DETAILS_NEW_ENTRY =
+      By.cssSelector("#activitiesAsCase #actionNewEntry");
+  public static final By AcC_START_OF_ACTIVITY = By.cssSelector(".v-window #startDate");
+  public static final By AcC_END_OF_ACTIVITY = By.cssSelector(".v-window #endDate");
+  public static final By AcC_DESCRIPTION = By.cssSelector(".v-window #description");
+  public static final By AcC_ACTIVITY_AS_CASE_TYPE =
+      By.cssSelector(".v-window #activityAsCaseType");
+  public static final By AcC_ROLE = By.cssSelector(".v-window #role");
+  public static final By AcC_TYPE_OF_PLACE = By.cssSelector(".v-window #typeOfPlace");
+  public static final By AcC_CONTINENT = By.cssSelector(".v-window #continent");
+  public static final By AcC_SUBCONTINENT = By.cssSelector(".v-window #subcontinent");
+  public static final By AcC_COUNTRY = By.cssSelector(".v-window #country");
+  public static final By AcC_REGION = By.cssSelector(".v-window #region");
+  public static final By AcC_DISTRICT = By.cssSelector(".v-window #district");
+  public static final By AcC_COMMUNITY = By.cssSelector(".v-window #community");
+  public static final By AcC_STREET = By.cssSelector(".v-window #street");
+  public static final By AcC_HOUSE_NUMBER = By.cssSelector(".v-window #houseNumber");
+  public static final By AcC_ADDITIONAL_INFORMATION =
+      By.cssSelector(".v-window #additionalInformation");
+  public static final By AcC_POSTAL_CODE = By.cssSelector(".v-window #postalCode");
+  public static final By AcC_CITY = By.cssSelector(".v-window #city");
+  public static final By AcC_AREA_TYPE = By.cssSelector(".v-window #areaType");
+  public static final By AcC_DETAILS = By.cssSelector(".v-window #details");
+  public static final By AcC_DONE_BUTTON = By.cssSelector(".v-window #commit");
+  public static final By AcC_DISCARD_BUTTON = By.cssSelector(".v-window #discard");
+
```

Comment:
Are we using these selectors ? if not then they should be removed.

Reply:
Also, let's add _OPTION if the locator represents the options

Reply:
These are leftovers from a merge I tried from George's branch on EpiDatafor Case. 
Remove the file.